cmd/neofs-lens/internal/meta/status.go

```diff
@@ -0,0 +1,48 @@
+package meta
+
+import (
+	"fmt"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var statCMD = &cobra.Command{
+	Use:   "stat",
+	Short: "Object status information",
+	Long:  `Get metabase's indexes related to an object.`,
+	Args:  cobra.NoArgs,
+	Run:   statFunc,
+}
+
+func init() {
+	common.AddAddressFlag(statCMD, &vAddress)
+	common.AddComponentPathFlag(statCMD, &vPath)
+}
+
+func statFunc(cmd *cobra.Command, _ []string) {
+	var addr oid.Address
+
+	err := addr.DecodeString(vAddress)
+	common.ExitOnErr(cmd, common.Errf("invalid address argument: %w", err))
+
+	db := openMeta(cmd, true)
+	defer db.Close()
+
+	res, err := db.ObjectStatus(addr)
+	common.ExitOnErr(cmd, common.Errf("reading object status: %w", err))
+
+	cmd.Printf("Object state: %s\n", res.State)
+	cmd.Printf("Starage's ID: %s\n", res.StorageID)
+	cmd.Println("Indexes:")
+	for _, bucket := range res.Buckets {
+		valStr := "<empty>"
+		if bucket.Value != nil {
+			valStr = fmt.Sprintf("%x", bucket.Value)
+		}
+
+		cmd.Printf("\tBucket: %d\n"+
+			"\tValue (HEX): %s\n", bucket.BucketIndex, valStr)
```

Comment:
`(HEX)` is incorrect for `<empty>`

Reply:
agree. do you have an alternative? first, it was just an empty string but i did not like it as a user when i saw it so it was added. also, it is quite clear that it (`<empty>`) is not a HEX string and it cannot be misread, imo